src/content/components/BugList/BugList.js

```diff
@@ -93,32 +162,223 @@ export class BugList extends React.PureComponent {
 
   renderFilters() {
     return (
-      <EditorGroup>
-        {this.props.showResolvedOption ? (
-          <span>
-            <input
-              type="checkbox"
-              onChange={this.onCheckShowResolved}
-              checked={this.state.showResolved}
-            />{" "}
-            Show Resolved
-          </span>
-        ) : null}
-      </EditorGroup>
+      <div>
+        <EditorGroup>
+          {this.props.showResolvedOption ? (
+            <span>
+              <input
+                type="checkbox"
+                onChange={this.onCheckShowResolved}
+                checked={this.state.showResolved}
+              />{" "}
+              Show Resolved
+            </span>
+          ) : null}
+        </EditorGroup>
+      </div>
     );
   }
 
+  onEditDropdownSelect(selectedBugs, selection) {
+    this.setState({ editSelection: selection.value });
+    if (selection.value == 2) {
+      window.location.href = this.getBulkEditLink(selectedBugs);
+    }
+
+    if (selectedBugs.length == 1) {
+      const selectedBug = this.props.bugs.find(
+        bug => bug.id == parseInt(selectedBugs[0])
+      );
+
+      if (selection.value == 0) {
+        const newIteration =
+          selectedBug.cf_fx_iteration == "---"
+            ? this.context.iterations.getIteration().number
+            : this.context.iterations.getAdjacentIteration(
+                1,
+                null,
+                lookupIterations([selectedBug.cf_fx_iteration])
```

Comment:
This method `lookupIterations` is for the server, not content. It is generated once per day and requested from the server by `/api/iterations` (see server.ts). The content then generates an IterationLookup from it, which is then passed down to all the components via React context. See other examples that use `iterations` from GlobalContext.